### PR TITLE
Update documentation related to start_test_network.sh to talk about sandbox.sh instead

### PR DESCRIPTION
**Pull request by achamayou**
Helps with #1700, but isn't complete. We should talk about JS app bundles, and we will once 0.14.2 is out of the door.

### CODE REVIEW

File: doc/developers/demo.rst [hunk 1]
```diff
@@ -18,27 +19,26 @@ This script automates the steps described in :doc:`/operators/start_network`, in
...
 
-The following command will run a simple 3-node test network on a single machine:
+The following command will run a simple one node test network on a single machine:
 
 .. code-block:: bash
 
     $ cd CCF/build
 
-    $ ../start_test_network.sh --package ./liblogging.enclave.so.signed
+    $ ../sandbox.sh -p ./liblogging.virtual.signed
```
Comment (letmaik) [inline doc/developers/demo.rst:28]:
```suggestion
    $ ../sandbox.sh -p ./liblogging.virtual.so
```

File: doc/developers/demo.rst [hunk 2]
```diff
@@ -18,27 +19,26 @@ This script automates the steps described in :doc:`/operators/start_network`, in
...
-    [15:27:46.644] See https://microsoft.github.io/CCF/master/users/issue_commands.html for more information.
-    [15:27:46.644] Press Ctrl+C to shutdown the network.
-
-The command output shows the addresses of the CCF nodes where commands may be submitted (eg, in this case, via ``curl https://127.251.192.205:36981/...``). The output and error logs of each node can be found in the node-specific directory in the workspace (eg, ``workspace/test_network_0/err`` is node 0's stderr).
+    [16:14:05.294] Starting 1 CCF nodes...
+    [16:14:05.295] Virtual mode enabled
+    [16:14:10.010] Started CCF network with the following nodes:
+    [16:14:10.011]   Node [0] = https://127.0.0.1:8000
+    [16:14:10.011] You can now issue business transactions to the ./liblogging.virtual.so application.
+    [16:14:10.011] Keys and certificates have been copied to the common folder: /data/amchamay/CCF/build/workspace/sandbox_common
```
Comment (letmaik) [inline doc/developers/demo.rst:36]:
```suggestion
    [16:14:10.011] Keys and certificates have been copied to the common folder: /data/src/CCF/build/workspace/sandbox_common
```

File: doc/quickstart/test_network.rst [hunk 3]
```diff
@@ -6,67 +6,67 @@ Starting a Test Network
...
-    [14:48:12.138] You can now issue business transactions to the ./liblogging.enclave.so.signed application.
-    [14:48:12.138] Keys and certificates have been copied to the common folder: /path/to/test_network_common
-    [14:48:12.138] See https://microsoft.github.io/CCF/master/users/issue_commands.html for more information.
-    [14:48:12.138] Press Ctrl+C to shutdown the network.
+    [16:14:05.294] Starting 1 CCF nodes...
+    [16:14:05.295] Virtual mode enabled
+    [16:14:10.010] Started CCF network with the following nodes:
+    [16:14:10.011]   Node [0] = https://127.0.0.1:8000
+    [16:14:10.011] You can now issue business transactions to the ./liblogging.virtual.so application.
+    [16:14:10.011] Keys and certificates have been copied to the common folder: /data/amchamay/CCF/build/workspace/sandbox_common
```
Comment (letmaik) [inline doc/quickstart/test_network.rst:26]:
```suggestion
    [16:14:10.011] Keys and certificates have been copied to the common folder: /data/src/CCF/build/workspace/sandbox_common
```

File: doc/developers/demo.rst [hunk 4]
```diff
@@ -18,27 +19,26 @@ This script automates the steps described in :doc:`/operators/start_network`, in
...
 
-The following command will run a simple 3-node test network on a single machine:
+The following command will run a simple one node test network on a single machine:
 
 .. code-block:: bash
 
     $ cd CCF/build
 
-    $ ../start_test_network.sh --package ./liblogging.enclave.so.signed
+    $ ../sandbox.sh -p ./liblogging.virtual.so
```
Comment (eddyashton) [inline doc/developers/demo.rst:28]:
```suggestion
    $ ../tests/sandbox.sh -p ./liblogging.virtual.so
```

`start_test_network.sh` was under the project root, and I think we also copied it into the build directory. Perhaps we should be doing that with `sandbox.sh` as well, but I think that can be implicit in the "make the build dir look like the install dir" task.

File: doc/quickstart/test_network.rst [hunk 5]
```diff
@@ -6,68 +6,68 @@ Starting a Test Network
...
+    [16:14:10.011]   Node [0] = https://127.0.0.1:8000
+    [16:14:10.011] You can now issue business transactions to the ./liblogging.virtual.so application.
+    [16:14:10.011] Keys and certificates have been copied to the common folder: /data/src/CCF/build/workspace/sandbox_common
+    [16:14:10.011] See https://microsoft.github.io/CCF/master/users/issue_commands.html for more information.
+    [16:14:10.011] Press Ctrl+C to shutdown the network.
 
 .. note::
 
-    - To use CCF `virtual` mode, the same command can be run with ``TEST_ENCLAVE=virtual`` set as environment variable and the virtual version of the enclave application passed to the script. For example ``$ TEST_ENCLAVE=virtual ../start_test_network.sh --package ./liblogging.virtual.so``.
+    - `sandbox.sh` defaults to using CCF's `virtual` mode, which does not require or make use of SGX. To load debug or release enclaves and make use of SGX, `-e` must be set to the right value, for example: `./sandbox.sh -e release -p ./liblogging.enclave.so.signed`
```
Comment (eddyashton) [inline doc/quickstart/test_network.rst:32]:
```suggestion
    - `sandbox.sh` defaults to using CCF's `virtual` mode, which does not require or make use of SGX. To load debug or release enclaves and make use of SGX, `--enclave-type` must be set to the right value, for example: `./sandbox.sh --enclave-type release -p ./liblogging.enclave.so.signed`
```

I prefer to use the long arg names where we have space. Also makes it easier to find this option (and its possible values) in the `--help` output.

**Review by eddyashton — APPROVED** [review]

**Pull request merged**